src/OpenTelemetry.Exporter.Geneva/Metrics/OtlpProtobuf/WireFormat.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+namespace OpenTelemetry.Exporter.Geneva;
+
+/// <summary>
+/// This class contains constants and helper functions useful for dealing with
```

Comment:
Add a link to the protobuf doc that mentions this encoding: https://protobuf.dev/programming-guides/encoding/#structure